views/treetest/view.ejs

```diff
@@ -8,8 +8,9 @@
 		<!-- custom css -->
 		<link rel="stylesheet" type="text/css" href="/css/treetest.css" />
 		<title>Kort Treetest</title>
-		<script type="text/javascript" src="/js/treetest.js"/></script>
+		<script type="text/javascript" src="/js/treetest.js"></script>
 		<script type="text/javascript" src="/js/treetest-view.js"></script>
+		<script type="text/javascript" src="/socket.io/socket.io.js"></script>
```

Comment:
Good point! What I've done to fix this is get the response ID from the `<input id="resid">` on line 25. When previewing, this will be set to `preview`, so preview runs are identifiable in the database and can be filtered out

Reply:
See [commit b70280](https://github.com/ScottLogic/kort/pull/7/commits/b70280a6111a8655d46932c118cadea5992ed768)